f600233d1b154d507b782f84604bcb12/update/sys_atf_step_6a417e931b5d0d907b782f84604bcb0f.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_atf_step">
+    <sys_atf_step action="INSERT_OR_UPDATE">
+        <active>true</active>
+        <copied_from/>
+        <description>Confirm that at least the following application menus are visible: Mission Owner</description>
+        <display_name>Application Menu Visibility</display_name>
+        <inputs/>
+        <mugshots_cache_json/>
+        <notes/>
+        <order>1</order>
+        <snapshot/>
+        <step_config display_value="Application Menu Visibility">6228cf753752030064a52f3c8e41f1a8</step_config>
+        <sys_class_name>sys_atf_step</sys_class_name>
+        <sys_created_by>prasanthi.alaparthi-ctr@ccpo.mil</sys_created_by>
```

Comment:
Yeah, this is surely something that needs to be worked out and tested to avoid this and have a decent dev workflow. I will try with the job table I great and see how that goes.